### PR TITLE
Create workflow to automatically make vw sized hero images

### PR DESCRIPTION
**Pull request by ryceg**
## What does this do?

So, this is *meant* to create a workflow that creates the correct sized images for the different viewports in the `gh-pages` branch. Unfortunately, I'm not very experienced with GitHub Actions, and don't know whether this will even come close to doing what I intend it to do. 

## How was this tested? Did you test the changes in the compiled `.html` file?

No testing, needs somebody else to look at it.

## Is there a [GitHub Issue](https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/issues?q=is%3Aopen+is%3Aissue) that this is resolving?

Nope.

### CODE REVIEW

File: .github/workflows/resize-images.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Resize Images
+
+on:
+  push:
+    branches:
+      - master
```
Comment (ryceg) [inline .github/workflows/resize-images.yml:6]: I think that it should only happen on releases, though I don't know how to implement that.

File: .github/workflows/resize-images.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: jruipinto/ImageMagick-action@v1
+        with:
+          # ImageMagick command to be executed
+          command: magick -path src/Resources/img/hero/*.jpg -auto-orient -resize x360 src/Resources/img/hero/*-x360.jpg
```
Comment (ryceg) [inline .github/workflows/resize-images.yml:16]: Not sure if there's a more performative way to do this.

File: .github/workflows/resize-images.yml (hunk 1 above)
Comment (Glinkis) [reply .github/workflows/resize-images.yml:6]: Well, what *is* a release? Do we always want to do one on merging to master?

Comment (ryceg) [reply .github/workflows/resize-images.yml:6]: Automating the versioning in the correct way would be a pain. I don't think we'd want to do that.

**Pull request merged**